src/PerformanceSensitiveAnalyzers/UnitTests/CallSiteImplicitAllocationAnalyzerTests.cs

```diff
@@ -147,5 +148,30 @@ public void SomeMethod()
                 // Test0.cs(12,9): warning HAA0102: Non-overridden virtual method call on a value type adds a boxing or constrained instruction
                 VerifyCS.Diagnostic(CallSiteImplicitAllocationAnalyzer.ValueTypeNonOverridenCallRule).WithLocation(12, 9));
         }
+
+        [Fact, WorkItem(3272, "https://github.com/dotnet/roslyn-analyzers/issues/3272")]
+        public async Task C()
+        {
+            await new VerifyCS.Test
+            {
+                ReferenceAssemblies = ReferenceAssemblies.NetFramework.Net45.Default,
+                TestCode = @"
+using System;
+using Roslyn.Utilities;
+
+public class MyClass
+{
+    [PerformanceSensitive(""uri"")]
+    public void Testing()
+    {
+        Params(); //no allocation
+    }
+
+    public void Params(params int[] args)
+    {
+    }
+}",
+            }.RunAsync();
```

Comment:
Test code look good to me, just need to call though `VerifyCS.VerifyAnalyzerAsync` which will include attribute source.

Reply:
The full form is required since we are changing the reference assemblies.